File: JAVA/temp.java
import java.util.Scanner;

public class temp {
    public static void main(String args[]){
        Scanner scanner = new Scanner(System.in);
        
        System.out.println("Qual a temperatura em Celcius?");
        double num1 = scanner.nextDouble();
        
        double resultado = ((num1 * 1.8) + 32);
        System.out.println(resultado);
    }
}

File: HASKELL/temp.hs
celcf :: Double -> Double
celcf temp = (temp * 1.8) + 32

main :: IO ()
main = do
    putStrLn "Qual a temperatura em Celcius? "
    num1 <- readLn :: IO Double
    let temp = num1
    let tempfin = celcf temp
    putStrLn ("A temperatura em Fahrenheit é: " ++ show tempfin)